documentation/dynamicHtml.md

```diff
@@ -0,0 +1,37 @@
+## 1. `newImageUpload(uid)`
```

Comment:
Disclaimer: I'm not the maintainer of this repo, just a curious user.

I love a good documentation; it helps to get into a project more smoothly.

Here, I wonder why you add source documentation in separate files. Why not write JSDoc in the source files directly? 🤔

Reply:
Hi @csemrau yes, the repo needed better documentation, it's been neglected for a while now.

I had previously started adding inline JSDocs in my local copy. I've quickly pushed some changes to branch `develop/revamp2024`, now that the project's getting some attention again 😅.

There are some more changes upcoming to improve code quality.